module/VuFind/src/VuFind/Auth/ILSAuthenticator.php

```diff
@@ -29,9 +29,8 @@
 
 namespace VuFind\Auth;
 
-use Laminas\Config\Config;
```

Comment:
I'm a little confused by the Laminas config changes showing up in this PR; that's a change made in the dev branch after this PR was opened. It looks like somehow some parts of those changes made it here, even though the whole upstream dev branch hasn't been merged in.

I think I'm going to pause my review until this is resolved, since it looks like the code is a little bit out of sync. Do you mind merging the upstream dev branch into this PR and pushing up those changes? I'll continue reviewing once everything is up to date again. :-)

Reply:
I believe this should be merged in as of 12/23/2024